shopware/paas-meta/6.4/.platform/applications.yaml

```diff
@@ -140,6 +141,9 @@
             if [ -e bin/setup-fastly.sh ]; then
                 ./bin/setup-fastly.sh
             fi
+            
+            # Kill php-fpm to clear the opcache
+            pkill -f -u "$(whoami)" php-fpm
```

Comment:
Can't we send USR2 to php-fpm to have the same result instead of killing the process?

Reply:
Cool, didn't know about that! I have updated it.